src/ResourceManager/RecoveryServices/AzureRM.RecoveryServices.psd1

```diff
@@ -9,7 +9,7 @@
 @{  
   
 # Version number of this module.  
-ModuleVersion = '1.0.6'
```

Comment:
Please revert this change. We are bumping up all the module versions during release.